build/serving/mlserver/Dockerfile

```diff
@@ -1,15 +1,17 @@
-FROM python:3.7-slim
+FROM openjdk:slim
+
+COPY --from=python:3.7-slim / /
 
 ENV MODEL_STORE /mnt/models
 
 RUN apt-get update && \
-    apt-get -y --no-install-recommends install \
-      libgomp1 unzip
+  apt-get -y --no-install-recommends install \
+  libgomp1 unzip
 
 # Use MLServer for serving, see https://github.com/SeldonIO/MLServer
 WORKDIR /workspace
-ADD https://github.com/SeldonIO/MLServer/archive/master.zip .
-RUN unzip master.zip && pip install MLServer-master/[all] && rm -r MLServer-master && rm master.zip
+ADD https://github.com/FogDong/MLServer/archive/master.zip .
```

Comment:
I'll fix it when the mlserver code is ready to be merged in upstream and kelver.

Reply:
Should we wait until the change is merged to kleveross?